documentation/library_guide/api_for_dpcpp_kernels/tested_standard_cpp_api.rst

```diff
@@ -367,8 +367,9 @@ libstdc++(GNU)                                Provided with GCC*-7.5.0, GCC*-9.3
 --------------------------------------------- ---------------------------------------------
 libc++(LLVM)                                  Provided with Clang*-11.0
 --------------------------------------------- ---------------------------------------------
-Microsoft Visual C++* (MSVC) Standard Library Provided with Microsoft Visual Studio* 2017
-                                              and Microsoft Visual Studio 2019
+Microsoft Visual C++* (MSVC) Standard Library Provided with Microsoft Visual Studio* 2017;
+                                              Microsoft Visual Studio 2019; and Microsoft 
```

Comment:
Should an asterisk follow every use of `Microsoft Visual Studio` to be consistent with the usage on the libstdc++ line?

Reply:
No, we just need an asterisk at the initial use, we are covered after that.

Reply:
Because Microsoft Visual C++ (MSVC) and Microsoft Visual Studio are two products, each need an asterisk on the first use. This did change from a few years ago, when we DID need an asterisk on each instance.